expected/index.out

```diff
@@ -55,6 +55,7 @@ SELECT count(*) FROM spheretmp4 WHERE l && scircle '<(1,1),0.3>';
 
 -- create idx
 CREATE TABLE spheretmp1b AS TABLE spheretmp1;
+ANALYZE spheretmp1;
```

Comment:
I know this is in the expected output, but I was wondering if the ANALYZE in its corresponding SQL file should come after the three CREATE INDEX statements instead of before? Does it matter? I would think it does?

Reply:
It does not matter, non-functional indexes (not on a expression) don't have stats on their own. For our purposes, the ANALYZE will just collect the number of pages/rows in that table and record it in pg_class.

Implementing actual stats for the pg_sphere datatypes would be another interesting feature to explore. "In which regions of the sky are the actual stars in this table?"